torch/distributions/transforms.py

```diff
@@ -500,8 +511,8 @@ def __eq__(self, other):
 
     @property
     def sign(self):
-        if isinstance(self.scale, numbers.Number):
-            return 1 if self.scale > 0 else -1 if self.scale < 0 else 0
+        if isinstance(self.scale, numbers.Integral):
```

Comment:
Hmm, changing `Number` to `Integral` here would lead to a crash if scale is float, right? (because there are no `sign` method for the float type, is there?